drivers/pds/api/accountRoleBinding.go

```diff
@@ -90,3 +90,17 @@ func (accountRoleBinding *AccountRoleBinding) AddUser(accountID string, email st
 	}
 	return nil
 }
+
+// RemoveUser function delete the user from given account.
+func (accountRoleBinding *AccountRoleBinding) RemoveUser(userID string) (*status.Response, error) {
+	client := accountRoleBinding.apiClient.AccountRoleBindingsApi
+	ctx, err := pdsutils.GetContext()
+	if err != nil {
+		return nil, fmt.Errorf("error in getting context for api call: %v", err)
+	}
+	res, err := client.ApiAccountsIdRoleBindingsDelete(ctx, userID).Execute()
+	if err != nil && res.StatusCode != status.StatusOK {
+		return nil, fmt.Errorf("error when calling `ApiBackupTargetsIdDelete``: %v", err)
```

Comment:
will there be a case where er !=nil and statuscode is ok?

Reply:
Updated